docs/source/en/chat_templating.md

```diff
@@ -196,7 +196,7 @@ Not all models require generation prompts. Some models, like LLaMA, don't have a
 special tokens before bot responses. In these cases, the `add_generation_prompt` argument will have no effect. The exact
 effect that `add_generation_prompt` has will depend on the template being used.
 
-## What does "continue_last_message" do?
+## What does "continue_final_message" do?
```

Comment:
The changes in this file are unrelated to the PR. If you intentionally added these lines, please open a separate PR 🤗 If not, rebasing may help